From 17a57029d0745e69214a033416a0fdd59920271b Mon Sep 17 00:00:00 2001
From: VirusABC <83475705+VirusABC@users.noreply.github.com>
Date: Wed, 7 May 2025 17:02:03 -0300
Subject: [PATCH] Incluir atalhos dummy R e Rstudio
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Cria entradas R e Rstudio no modo gráfico abrindo o tutorial respectivo na wiki
---
 dev-x/tasks/main.yml | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/dev-x/tasks/main.yml b/dev-x/tasks/main.yml
index 14cff13..53e2839 100644
--- a/dev-x/tasks/main.yml
+++ b/dev-x/tasks/main.yml
@@ -33,3 +33,21 @@
   apt:
     update_cache: yes
     pkg: codium
+
+- name: copia os icones do rstudio e do R nos locais apropriados
+  ansible.builtin.copy:
+    src: icons/
+    dest: /usr/share/icons
+    owner: root
+    group: root
+    directory_mode: '755'
+    mode: '644'
+
+- name: copia os .desktop de R e Rstudio para redirecionar à página da wiki
+  ansible.builtin.copy:
+    src: applications/
+    dest: /usr/share/applications
+    owner: root
+    group: root
+    directory_mode: '755'
+    mode: '644'
